pkgs/applications/audio/mbrola/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkgconfig }:
```

Comment:
CI seems to fail because you are requesting pkgconfig instead of pkg-config. But most of these are not used anyway.

Reply:
thanks, fixed

i think i added all those because they are needed for the build to succeed. i don't remember...

Reply:
Well, you are not using them in inputs any more so they can be removed.

Reply:
you are right, i tried it